requirements.txt

```diff
@@ -32,11 +32,10 @@ sentencepiece>=0.2.0
 tokenizers>=0.11.1
 toml>=0.10.2
 # Default version. Can be overridden in extra requirements
-torch>=2.3.0,<2.6.0
+torch>=2.3.0,<2.7.0
```

Comment:
Same here, please reconsider the upper version cap.

Also is anybody testing that InstructLab still works with `torch==2.3.0`? If not, then you should raise the lower cap.

Reply:
Is it true? I don't think we ever tested lowest versions before. It would be great to have this kind of testing, but in its absence, I think it's fine to keep caps conservative and bump them only when we know of a particular incompatibility. Am I thinking about it in a wrong way?

Reply:
We know that the version of vllm being used now only works with torch 2.6, for example. So saying that instructlab could work with 2.3 is clearly wrong.

Reply:
If we *know* then sure let's bump.

Reply:
Changed

```
torch>=2.3.0,<2.7.0
```

to 

```
torch>=2.6.0,<2.7.0
```

just now